### ISSUE TITLE
`cuckoo_generator version` prints empty line

### ISSUE THREAD

**Issue by davidcairns** [Brightify/Cuckoo#198]
The other commands work fine, but I’m not able to get the version. It just prints a blank line!
```
$ ./Pods/Cuckoo/cuckoo_generator 
Available commands:

   generate   Generates mock files
   help       Display general or command-specific help
   version    Prints the version of this generator.
$ ./Pods/Cuckoo/cuckoo_generator version

$ ./Pods/Cuckoo/cuckoo_generator --version # ???
Unrecognized command: '--version'. See `./Pods/Cuckoo/cuckoo_generator help`.
```

Thanks!

**Comment by MatyasKriz** [comment]
Hey there! Thanks for the report, indeed it seems that someone forgot to implement the version option! If you'd like to create a PR, feel free to do so, I'll take a look at it tomorrow, but I would be happy to see your PR waiting if you're willing. 🙂

**Comment by davidcairns** [comment]
Hey, thanks. I would love to get to it myself! But I just moved so I’m going a little nuts right now. Seems like an easy intro to your codebase though, so maybe I’ll try to knock it out when I have a little time this week 😄👍

**Comment by MatyasKriz** [comment]
That would be cool! I'm sorry for not taking a look into it when I promised, I'm kinda busy with school right now, so your PR will really help me out here. 🙂 Beware though that `Cuckoo` and `Cuckoo Generator` are two different things and this concerns the `Cuckoo Generator` I think. It's just as important, but the black magic is mostly hidden in the `Cuckoo` project. 😀

**Comment by MatyasKriz** [comment]
Fixed in 2.0. 🙂
